contracts/src/BeefyClient.sol

```diff
@@ -390,49 +417,57 @@ contract BeefyClient {
         }
     }
 
-    /**
-     * @dev Calculate minimum number of required signatures for the current validator set.
-     *
-     * This function approximates f(x) defined below for x in [1, 21_846):
-     *
-     *  x <= 10: f(x) = x * (2/3)
-     *  x  > 10: f(x) = max(10, ceil(log2(3 * x))
-     *
-     * Research by W3F suggests that `ceil(log2(3 * x))` is a minimum number of signatures required to make an
-     * attack unfeasible. We put a further minimum bound of 10 on this value for extra security.
-     *
-     * If the session has less than 10 active validators it's definitely some sort of local testnet
-     * and we use different logic (minimum 2/3 + 1 validators must sign).
-     *
-     * One assumption is that Polkadot/Kusama will never have more than roughly 20,000 active validators in a session.
-     * As of writing this comment, Polkadot has 300 validators and Kusama has around 1000 validators,
-     * so we are well within those limits.
-     *
-     * In any case, an order of magnitude increase in validator set sizes will likely require a re-architecture
-     * of Polkadot that would make this contract obsolete well before the assumption becomes a problem.
-     *
-     * Constants generated with the help of scripts/minsigs.py
-     */
-    function minimumSignatureThreshold(uint256 validatorSetLen) internal pure returns (uint256) {
-        if (validatorSetLen <= 10) {
+    // Calculates the number of signature samples required by validator set length and the number of times a validator
+    // signature was used.
```

Comment:
Should clarify what "number of times a validator signature was used" actually means in relation to `signatureUseCount`. Which signature is being used multiple times where?

Reply:
Addressed in https://github.com/Snowfork/snowbridge/pull/956/commits/17d468f0427d9e040b1d8438d9202045834b6fe5.